.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,29 @@
+name: codeql
+
+# FIXME uncomment conditions before merging PR
+on:
+  push:
+    # branches:
+    # - master
+  pull_request:
+    # branches:
+    # - master
+  # schedule:
+  #   - cron: "0 3 * * 4"
```

Comment:
Is the intention to comment the above cron line? If yes, why it is not sufficient to run this just on PRs?

Reply:
Yes. The CodeQL scanner is evolving and gets new checks added to it from time to time. So, even if the code hasn't changed, the scanner might have done, so we should re-run the scanner.